spec/support/snap_application_form_helper.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module SnapApplicationFormHelper
+  def fill_in_name_and_birthday
+    fill_in "What is your first name?", with: "Jessie"
+    fill_in "What is your last name?", with: "Tester"
+    fill_in_birthday
+  end
+
+  def fill_in_birthday
+    select "January", from: "step_birthday_2i"
+    select "1", from: "step_birthday_3i"
+    select "1969", from: "step_birthday_1i"
+  end
+
+  def select_address_same_as_home_address
+    select_radio(
+      question: "Is this address the same as your home address?",
+      answer: "Yes",
+    )
+  end
+
+  def answer_household_more_info_questions(js: true)
```

Comment:
__not blocking__ - This is starting to smell like we may want a page object?

Reply:
good idea! I think this PR is a good first step in refactoring our feature specs because they were getting a bit difficult to read in that one big file, but I can definitely see us doing that in the near future! adding a trello card